catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -617,12 +593,13 @@ public void dispose() {
 		disposeClonedSprites();
 	}
 
-	public boolean makeManualScreenshot() {
+	public boolean makeScreenshot(boolean manual) {
```

Comment:
since we need an updated automatic screenshot everytime the stage finished/paused. so I modified this method and reused for taking the automatic- and the manual- screenshot.